src/app/places/places.component.html

```diff
@@ -1,22 +1,78 @@
-<div class="container">
-  <ul class="places">
-  <li data-test="list-places" *ngFor="let place of PlacesFromService | paginate: { itemsPerPage: 5, currentPage: page }">
-  <div class="img">
-  <img class="images" src="../../assets/images/{{ place.Id }}.jpg" />
-  </div>
-  <div class="text">
-  <h3>{{ place.Name }}</h3>
-  {{ place.Description }}
-  <h5>
-  {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
-  <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a
-  ><br />Tagi: {{ place.Tag }}<br />
-  Max ilość osób: {{ place.Max_persons }}
-  </h5>
-  <br />
+<div class="page">
+  <div class="sort-buttons">
+    <button
+      date-test="sort-button A-Z"
+      mat-raised-button
+      color="primary"
+      (click)="asc()"
+      class="sorting-button"
+    >
+      A-Z Sort
+    </button>
+    <button
+      date-test="sort-button Z-A"
+      mat-raised-button
+      color="primary"
+      (click)="desc()"
+      class="sorting-button"
+    >
+      Z-A Sort
+    </button>
+    <button
+      date-test="sort-button No Sort"
+      mat-raised-button
+      color="primary"
+      (click)="reset()"
+      class="sorting-button"
+    >
+      No Sort
+    </button>
   </div>
-  </li>
+  <ul class="places">
+    <li
+      data-test="list-places"
+      *ngFor="
+        let place of PlacesFromService
+          | paginate: { itemsPerPage: 5, currentPage: page }
+      "
+    >
+      <mat-card class="place-card">
+        <mat-card-header>
+          <mat-card-title>{{ place.Name }}</mat-card-title>
+        </mat-card-header>
+
+        <img
+          mat-card-image
+          class="images"
+          src="../../assets/images/{{ place.Id }}.jpg"
+        />
+        <mat-card-content class="content">
+          <div class="description">{{ place.Description }}</div>
+          <div class="info">
+            {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
+            <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a>
+            <br />
+            Max ilość osób: {{ place.Max_persons }}
+            <mat-chip-list
+              ><mat-chip
+                *ngFor="let tag of place.Tag"
+                color="primary"
+                selected
+                class="tag"
+              >
+                {{ tag }}
+              </mat-chip></mat-chip-list
+            >
+          </div>
+        </mat-card-content>
+      </mat-card>
+    </li>
   </ul>
   <pagination-controls (pageChange)="page = $event"></pagination-controls>
-  </div>
-  
\ No newline at end of file
+</div>
+<footer>
+  Autorzy:
+  <address>Rafał Pietrek (ratattuja@gmail.com)</address>
+  <address>Lucyna Książek-Sperka (lks@hoga.pl)</address>
+  Team Events ©
+</footer>
```

Comment:
In my opinion, footer should be located in another component

Reply:
done